components/homme/CMakeLists.txt

```diff
@@ -159,11 +163,14 @@ IF (HOMME_ENABLE_COMPOSE)
   SET (HOMME_USE_KOKKOS TRUE)
 ENDIF ()
 
-IF (HOMME_USE_CXX)
-  MESSAGE(STATUS "This configuration of HOMME requires a C++ compiler")
-  ENABLE_LANGUAGE(CXX)
-  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-ENDIF ()
+IF (HOMME_STANDALONE)
+  # Homme needs to setup its own flags
+  IF (HOMME_USE_CXX)
+    MESSAGE(STATUS "This configuration of HOMME requires a C++ compiler")
+    ENABLE_LANGUAGE(CXX)
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I think this might be not right for standalone either. Kokkos uses c++14, and either way, the standard should be set via the apposite cmake commands, rather than stuffed in the flags.

But there's no need to fix it here, so it's fine.

Reply:
I agree, this is a hacky approach. I will let homme devs decide what to do.